spec/models/placement_spec.rb

```diff
@@ -62,4 +62,32 @@
       end
     end
   end
+
+  describe "order_by_school_ids" do
```

Comment:
Are we ordering by the school ids because they've been sorted alphabetically?

Reply:
We are sorting by school ids because you can't sort them by 'schools.distance'. As distance isn't really a thing